.github/workflows/update-version.yml

```diff
@@ -10,39 +10,62 @@
 # governing permissions and limitations under the License.
 #
 
-name: Update Version
+name: Update Versions
+
 on:
   workflow_dispatch:
     inputs:
-      version: 
-        description: 'New version to use for the Edge extension. Example: 3.0.0'
+      version:
+        description: 'New version to use for the extension (ex: 1.2.3).'
+        type: string
+        required: true
+
+      branch:
+        description: 'The target branch where the version update will be applied and the pull request will be merged into.'
+        type: string
         required: true
 
       core-dependency:
-        description: '[Optional] Update Core dependency in pom.xml. Example: 3.0.0'
+        description: | 
+          If a version is provided, update AEPCore dependency in gradle.properties (ex: 1.2.3).
+          This is a published dependency in the pom.xml file.
+        type: string
+        required: false
+        default: ''
+
+      edge-identity-dependency:
+        description: | 
+          If a version is provided, update AEPEdgeIdentity dependency in gradle.properties (ex: 1.2.3).
+          This is a published dependency in the pom.xml file.
+        type: string
+        required: false
+        default: ''
+
+      edge-consent-dependency:
```

Comment:
We did not have consent dependency in iOS, Is consent a hard dependency for edge like Identity?